front/app/components/ConsentManager/Container.test.tsx

```diff
@@ -1,297 +0,0 @@
-import React from 'react';
-
-import eventEmitter from 'utils/eventEmitter';
-import { render, act, fireEvent } from 'utils/testUtils/rtl';
-
-// component to test
-import Container from './Container';
-import { CategorizedDestinations } from './typings';
-
-// mock utilities
-
-jest.mock('utils/cl-router/Link', () => ({ children }) => (
-  <button>{children}</button>
-));
-
-let updatePreference: jest.Mock;
-let resetPreferences: jest.Mock;
-let saveConsent: jest.Mock;
-let accept: jest.Mock;
-let reject: jest.Mock;
-
-const initialPreferences = {
-  analytics: undefined,
-  functional: undefined,
-  advertising: undefined,
-};
-
-const categorizedDestinations: CategorizedDestinations = {
-  analytics: ['google_analytics'],
-  functional: ['intercom'],
-  advertising: [],
-};
-
-const emptyFunction = () => {};
-
-describe('<Container />', () => {
-  beforeEach(() => {
-    updatePreference = jest.fn();
-    resetPreferences = jest.fn();
-    saveConsent = jest.fn();
-    accept = jest.fn();
-    reject = jest.fn();
-  });
-
-  it('renders correctly when no destinations are allowed by the tenant (i.e. isConsentRequired === false)', () => {
-    const { container } = render(
-      <Container
-        updatePreference={updatePreference}
-        resetPreferences={resetPreferences}
-        saveConsent={saveConsent}
-        accept={accept}
-        reject={reject}
-        isConsentRequired={false}
-        preferences={initialPreferences}
-        categorizedDestinations={categorizedDestinations}
-        onToggleModal={emptyFunction}
-      />
-    );
-
-    expect(
-      container.querySelector('#e2e-preference-dialog')
-    ).not.toBeInTheDocument();
-    expect(
-      container.querySelector('#e2e-cookie-banner')
-    ).not.toBeInTheDocument();
-
-    // wih no destinations allowed, isConsentRequired will be false so no banner
-    // but the modal is still accessible trough the cookie policy
-    act(() => {
-      eventEmitter.emit('openConsentManager');
-    });
-
-    expect(
-      container.querySelector('#e2e-preference-dialog')
-    ).toBeInTheDocument();
-    expect(
-      container.querySelector('#e2e-cookie-banner')
-    ).not.toBeInTheDocument();
-  });
```

Comment:
Already covered by the "logged out, cookie exists" block in ConsentManager.test.tsx